layers/roads.yaml

```diff
@@ -56,8 +56,8 @@ layers:
 
         highway:
             filter:
-                - kind: highway
-                - $zoom: { min: 8 }
+                kind: highway
+                $zoom: { min: 8 }
             highway-z8:
```

Comment:
@nvkelso is the intent here that the filter is true when `kind: highway` **AND** zoom >= 8 (what the PR changes to), or when either `kind: highway` **OR** zoom >= 8 (what it was before)?